### ISSUE TITLE
Vue 3.4.20 and up: pallete doesn't remember the selected color anymore and the pointer is no longer contained inside it

### ISSUE THREAD

**Issue by renszee** [gVguy/vue-color-input#5]
Hello,

Starting from Vue version 3.4.20 the color picker is having some issues:

- The saturation pointer no longer is contained inside the pallete (not centered to mouse click).
- The pallete doesn't automatically select the selected color anymore when opening, instead if resets to the bottom left.

I've reproduced it in this sandbox: https://codesandbox.io/p/sandbox/elated-tdd-mp3dy6

**Comment by renszee** [comment]
It seems to work again when you click on the box when it's already open, it then opens again but this time works like normal.
It looks like the props are being ignored the first time you click.

**Comment by renszee** [comment]
Ok, managed to make it work again with a workaround by adding a ref and forcefully triggering the init() function on the picker after a short delay when it opens:

```vue
<ColorInput
  ref="colorInput"
  v-model="pickedColor"
  @pick-start="opencolorPicker"
/>
``` 

```ts
const colorInput = ref()
const opencolorPicker = () => {
  const picker = colorInput.value.$refs.picker
  setTimeout(() => {
    picker.init()
  }, 50)
}
```

**Comment by gVguy** [comment]
Thanks for a detailed report!

The bug is now fixed in 1.0.9
